backend/maelstro/metadata/meta.py

```diff
@@ -147,8 +175,24 @@ def __init__(self, zipfile: bytes):
 
         super().__init__(xml_bytes, schema)
 
-    def update_geoverver_urls(self, mapping: dict[str, list[str]]) -> tuple[str, str]:
-        super().update_geoverver_urls(mapping)
+    def replace_geoserver_src_by_dst_urls(
```

Comment:
Is this still used even with XSLT ?

Reply:
removed

Reply:
well, on second thought, maybe the string replacement remains useful if there are no xslt transformations (for other clients...)
I'd maybe just leave it there